### PR TITLE
expand frequently used quick links

### PR DESCRIPTION
**Pull request by mayim-chayim**
rewritten contentScript.js into a loop

### CODE REVIEW

**Comment by mayim-chayim** [comment]
@kavhayushar please review 
Thanks

File: src/expandQuickLinks.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+let navMain = document.getElementById('nav-main');
+let quickLinks = document.getElementById('quick-links');
+
+navMain.appendChild(quickLinks.getElementsByClassName('icon-search-self')[0]);
+navMain.appendChild(quickLinks.getElementsByClassName('icon-search-new')[0]);
```
Comment (Sunegi) [inline src/expandQuickLinks.js:5]: Will fail if user isn't signed in;, perhaps use absolute URLs for all.

Comment (mayim-chayim) [reply src/expandQuickLinks.js:5]: correct
I will fix it once the settings page is done
and I have feedback on the second style I have posted

Thanks for your review

**Comment by kavhayushar** [comment]
Looks good. 

As @Sunegi pointed out that it throws an error when not logged in. This type of errors is actually found many more places in this project. Will need one PR to fix all of such errors.

**Review by kavhayushar — APPROVED** [review]

**Comment by mayim-chayim** [comment]
> Looks good.
> 
> As @Sunegi pointed out that it throws an error when not logged in. This type of errors is actually found many more places in this project. Will need one PR to fix all of such errors.

where?
the missing post button element is already fixed

**Comment by kavhayushar** [comment]
@mayim-chayim I now see that you already fixed them. Cannot find anymore of them. Thank you very much.

**Comment by mordechairoth** [comment]
@mayim-chayim  Please resolve the conflicts

**Comment by mayim-chayim** [comment]
> @mayim-chayim Please resolve the conflicts

This PR is not ready to merge 
we need a settings page before we can implement it

**Pull request closed without merging**